src/js/components/covid19/DataSourcesAndMethodologiesPage.jsx

```diff
@@ -20,12 +23,15 @@ import Footer from 'containers/Footer';
 import Header from 'containers/shared/HeaderContainer';
 import { useDefCodes } from 'containers/covid19/WithDefCodes';
 import MetaTags from 'components/sharedComponents/metaTags/MetaTags';
-import StickyHeader from 'components/sharedComponents/stickyHeader/StickyHeader';
 import { stickyHeaderHeight } from 'dataMapping/stickyHeader/stickyHeader';
 import { getStickyBreakPointForSidebar, useDynamicStickyClass } from 'helpers/stickyHeaderHelper';
 import Sidebar from 'components/sharedComponents/sidebar/Sidebar';
 import ExternalLink from 'components/sharedComponents/ExternalLink';
-import ShareIcon from 'components/sharedComponents/stickyHeader/ShareIcon';
+
+const getEmailSocialShareData = {
+    subject: "COVID-19 Spending: Data Sources & Methodology",
+    body: `View COVID-19 Spending: Data Sources & Methodology on USAspending.gov: ${"https://www.usaspending.gov/#/disaster/covid-19/data-sources"}`
```

Comment:
This email body isn't showing up for me either.. let me know if these are working for you!